superset-frontend/package.json

```diff
@@ -208,6 +208,7 @@
     "rison": "^0.1.1",
     "scroll-into-view-if-needed": "^2.2.28",
     "shortid": "^2.2.6",
+    "source-map": "^0.7.4",
```

Comment:
Shouldn't this be a dev dependency?

Reply:
I feel like I tried that and it didn't work for some reason. The sourcemaps are generated/used by prod builds created by `npm run build` so maybe it's used in rendering/utilizing the source maps in those builds? I'm not exactly sure, but I somewhat recall it throwing the same error when it was a dev dependency.

Reply:
Ok, it's now a dev dependency :D Seems to work!

Reply:
You were right... n`pm run dev`, `npm run dev-server`, and `npm run buid` all seem to work fine for me with it as a dev dependency. Game on!